### PR TITLE
Add dictionary definition for "Flowchart"

### PR DESCRIPTION
**Pull request by nik132-eng**
## This PR fixes...

This PR fixes the Dictionary definition of "Flowcahrt"

## What I did...

- Add dictionary definition for "Flowchart".
- Add productmanager's perspective.

## How to test...

_In a list format, tell us how to test this PR._
Ex:

1. Navigate to /Dictionary
2. Search "Flowchart" 
3. See the definition and perspective show up under "Flowchart"
4. Check grammar and typos

## I learned...

learnt perfect definition

### CODE REVIEW

File: src/assets/content/dictionary/flowchart.md [hunk 1]
```diff
@@ -1,6 +1,9 @@
 ---
 title: Flowchart
-definition:
-sources: []
-perspectives: []
+definition: a diagram that shows step-by-step progression through a procedure or system especially using connecting lines and a set of conventional symbols
+sources:
+- sourceurl: https://www.merriam-webster.com/dictionary/flowchart
+perspectives:
+- meaning: which step/functionality work after which step. this roadmap is flowchart.
```
Comment (novellac) [inline src/assets/content/dictionary/flowchart.md]:
```suggestion
- meaning: which step/functionality comes after which step. A roadmap is a flowchart
```
suggestion: I've made suggestion for clarity and syntax for the perspective. You can use the Github UI to accept the suggestion here, or make an extra commit to this PR.

**Review by novellac — CHANGES_REQUESTED** [review]
Almost there! I've left one suggestion, and I'll be happy to approve and merge after that.

**Comment by nik132-eng** [comment]
yes can you check this now?

**Review by novellac — APPROVED** [review]
Looks great! Thanks for the Hacktoberfest contribution 🍁

**Pull request merged**